### PR TITLE
(#300) Wrap `is_private_session` flag 

### LINKED ISSUE #300: `Device` does not wrap the `is_private_session` flag.
The API will withhold information about playback if the client is in a private session.

Typically you can check for this from the `device` object from the playback endpoint, but the wrapper does not have the flag.

Below are some refs showing the flag in the responses.

https://developer.spotify.com/documentation/web-api/reference/#/operations/get-information-about-the-users-current-playback
https://developer.spotify.com/documentation/web-api/reference/#/operations/get-a-users-available-devices

### PR DESCRIPTION
**Pull request by jdngray77**
fixes #300 

- Device includes flag, it's getter, setter, and factory.
- Test data modified to include the flag.
- All existing tests pass.

Functionality tested live :

https://user-images.githubusercontent.com/50697488/184509305-2b3ec99a-6685-4269-94a3-9e04aa375425.mov

### CODE REVIEW

File: src/main/java/se/michaelthelin/spotify/model_objects/miscellaneous/Device.java [hunk 1]
```diff
@@ -46,6 +48,16 @@ public Boolean getIs_active() {
...
+  /**
+   * Get whether the device is a private session. Access to playback info is limited on private session clients.
+   *
+   * @return whether the user has put the client into a private session.
+   */
+  public Boolean getIs_private_session() {
+    return is_private_session;
+  }
+
+
```
Comment (dargmuesli) [inline src/main/java/se/michaelthelin/spotify/model_objects/miscellaneous/Device.java:60]:
```suggestion
  /**
   * Get whether the device is in a private session. Access to playback info is limited on clients in a private session.
   *
   * @return Whether the user has put the client into a private session.
   */
  public Boolean getIs_private_session() {
    return is_private_session;
  }

```

File: src/main/java/se/michaelthelin/spotify/model_objects/miscellaneous/Device.java [hunk 2]
```diff
@@ -126,6 +139,17 @@ public Builder setIs_active(Boolean is_active) {
       return this;
     }
 
+    /**
+     * The private session state setter.
+     *
+     * @param is_private_session If this device is the currently active device.
```
Comment (dargmuesli) [inline src/main/java/se/michaelthelin/spotify/model_objects/miscellaneous/Device.java:145]:
```suggestion
     * @param is_private_session If this device is currently in a private session.
```

**Review by dargmuesli — CHANGES_REQUESTED** [review]

**Comment by jdngray77** [comment]
My bad, i should've proof-read those.

**Review by dargmuesli — APPROVED** [review]

**Comment by dargmuesli** [comment]
Should be released in `v7.2.0` shortly.

**Pull request merged**